README.md

```diff
@@ -20,18 +20,18 @@ This action will create a github release and optionally upload an artifact to it
 - **omitNameDuringUpdate**: Indicates if the release name should be omitted during updates. The name will still be applied for newly created releases. This will preserve the existing name during updates.
 - **owner**: Optionally specify the owner of the repo where the release should be generated. Defaults to current repo's owner.
 - **prerelease**: Optionally marks this release as prerelease. Set to true to enable.
-- **replacesArtifacts**: Indicates if existing release artifacts should be replaced. Defaults to true.
+- **replacesArtifacts**: Indicates if existing release artifacts should be replaced. Defaults to `true`.
 - **repo**: Optionally specify the repo where the release should be generated. Defaults to current repo.
 - **tag**: An optional tag for the release. If this is omitted the git ref will be used (if it is a tag).
-- **token**: (**Required**) The Github token. Typically this will be `${{ secrets.GITHUB_TOKEN }}`.
+- **token**: (**Required**) The GitHub token. Typically this will be `${{ secrets.GITHUB_TOKEN }}`.
 
 ## Action Outputs
 - **id**: The identifier of the created release.
 - **html_url**: The HTML URL of the release.
 - **upload_url**: The URL for uploading assets to the release.
 
 ## Example
-This example will create a release when tag is pushed:
+This example will create a release when a new tag is pushed:
```

Comment:
Could we revert this line? The workflow in this example will actually run when any update to a tag is pushed, whether it be a new tag or an existing tag.

Reply:
```suggestion
This example will create a release when a tag is pushed:
```